test_runner/fixtures/neon_fixtures.py

```diff
@@ -542,7 +542,7 @@ class S3Storage:
     access_key: str
     secret_key: str
     endpoint: Optional[str] = None
-    prefix_in_bucket: Optional[str] = None
+    prefix_in_bucket: Optional[str] = ""
```

Comment:
let's revert and see if CI fails without this change

```suggestion
    prefix_in_bucket: Optional[str] = None
```